cmd/daemon/main.go

```diff
@@ -589,6 +589,11 @@ func main() {
 	if err := rpc.StopKillSwitch(); err != nil {
 		log.Println(internal.ErrorPrefix, "stopping KillSwitch:", err)
 	}
+	if internal.FileExists(internal.DaemonPid) {
```

Comment:
Please move this logic to `internal.ManualListener`. `pidFilePath` can be an argument to that function.

Reply:
Done